service/src/amqp.ts

```diff
@@ -521,4 +541,6 @@ export class Amqp extends EventEmitter implements Client.Transport {
       }
     });
   }
+
+
```

Comment:
nit: blank lines.

Reply:
Done.

---
In reply to: [222796294](https://github.com/Azure/azure-iot-sdk-node/pull/372#discussion_r222796294) [](ancestors = 222796294)